pkg/graph/schema.graphql

```diff
@@ -1608,6 +1608,7 @@ type Query {
   urls(cedarSystemId: String!): [CedarURL!]!
   cedarSystemDetails(cedarSystemId: String!): CedarSystemDetails
   systemIntakeContacts(id: UUID!): SystemIntakeContactsPayload!
+  relatedSystemIntakes(id: UUID!): [UUID!]!
```

Comment:
Rather than returning UUIDs, perhaps it'd make sense to return [the SystemIntake type itself](https://github.com/CMSgov/easi-app/blob/b251e3e09e73331d80acf1d4ff74064beefc0e95/pkg/graph/schema.graphql#L934).

This would allow the frontend to query these related Intakes and easily get their names, and any other related info, without having to make another query by ID

Reply:
Made this change